http4s-server/src/main/scala/com/avast/server/toolkit/http4s/Http4sServerModule.scala

```diff
@@ -0,0 +1,39 @@
+package com.avast.server.toolkit.http4s
+
+import java.net.{InetSocketAddress, StandardSocketOptions}
+
+import cats.effect.{ConcurrentEffect, Resource, Timer}
+import org.http4s.HttpApp
+import org.http4s.server.Server
+import org.http4s.server.blaze.BlazeServerBuilder
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration.Duration
+import scala.language.higherKinds
+
+object Http4sServerModule {
+
+  /** Makes [[org.http4s.server.Server]] initialized with the given config and [[org.http4s.HttpApp]]. */
+  def make[F[_]: ConcurrentEffect: Timer](config: Http4sServerConfig,
```

Comment:
I would prefer `makeBlazerServer` name.

Reply:
there are no other `make`like methods, so there's no need to disambiguate

Reply:
to express myself more clearly: I think this module should be dedicate to Blaze server specifically, so the object itself should be called `Http4sServerBlazeModule`. Than the method can be called just `make`